vhdlparser/VhdlParserErrorHandler.hpp

```diff
@@ -22,13 +22,15 @@ class VhdlErrorHandler: public ErrorHandler
     {
       warn(m_fileName,actual->beginLine,"syntax error '%s'",actual->image.data());
       error_count++;
+      exit(0);
```

Comment:
We have here a warning following by terminating doxygen (even with an OK code), I think it would be better to call instead of `warn` the function `term` (might be that either the lines should be a bit reformulated or and extra `term` function should be defined with the 2 extra argunments, filename and linenumber)

Reply:
removed exit(0) lines from file